### ISSUE TITLE
[Error] Occurs with `cancel_event = 'move'` When Pasting Twice

### ISSUE THREAD

**Issue by tummetott** [gbprod/yanky.nvim#154]
Hello,

When I set `cancle_event = 'move'` and I paste twice with yanky, the plugin throws an error.

### Reproduce:

1. Copy paste this minimal conf `mini.lua`:
```lua
local lazypath = "/tmp/lazy/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
    vim.fn.system { "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", "--branch=stable", lazypath }
end
vim.opt.rtp:prepend(lazypath)
vim.opt.termguicolors = true

require("lazy").setup ({
    {
        "gbprod/yanky.nvim",
        opts = {
            ring = {
                cancel_event = 'move',
            },
        }
    },
}, { root = "/tmp/lazy" })

vim.keymap.set({"n","x"}, "p", "<Plug>(YankyPutAfter)")
vim.keymap.set({"n","x"}, "P", "<Plug>(YankyPutBefore)")
vim.keymap.set({"n","x"}, "gp", "<Plug>(YankyGPutAfter)")
vim.keymap.set({"n","x"}, "gP", "<Plug>(YankyGPutBefore)")
vim.keymap.set("n", "<c-n>", "<Plug>(YankyCycleForward)")
vim.keymap.set("n", "<c-p>", "<Plug>(YankyCycleBackward)")
vim.keymap.set({"n","x"}, "y", "<Plug>(YankyYank)")
```
2. Run: `nvim --clean -u mini.lua`
3. Press: `i` --> `hallo` --> `ESC` --> `yiw` (write hallo and yank the word)
4. Press: `p` --> `p` (paste twice)

### Error message:
```
Error executing vim.schedule lua callback: /tmp/lazy/yanky.nvim/lua/yanky.lua:125: attempt to index field 'state' (a nil value)
stack traceback:
        /tmp/lazy/yanky.nvim/lua/yanky.lua:125: in function </tmp/lazy/yanky.nvim/lua/yanky.lua:124>
```

**Comment by gbprod** [comment]
Thanks for this issue report, I think I've foud a fix #156 :)
I'll merge it soon, Let me know if it's ok for you

**Comment by tummetott** [comment]
Amazing thank you for the fix
